exasol/toolbox/nox/_release.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import annotations
 
 import argparse
+import re
+import subprocess
+import sys
```

Comment:
nit: `sys` and  a few of the other imports at the top are unused. Please remove them.

tip: If you use an IDE, like one fromJetbrains, when you save a file you can choose to use something like https://www.jetbrains.com/guide/tips/optimize-imports/ & check the box to touch the imports.

aside: I'll try to write up a feature where we use `autoflake` to detect & handle some of the these things. It'll resolve quite a few of the linting issues before we turn that on in the CI pipelines.

* I use nit to denote things that are nitpicking. In many cases, these are optional and not critical for getting an approval, but I would appreciate them being done.